.github/workflows/python-docs-theme.yml

```diff
@@ -0,0 +1,118 @@
+name: Update and pull python-docs-theme project
+
+on:
+  push:
+    branches:
+      - 'main'
+    paths-ignore:
+      - 'docs/**'
+      - '.readthedocs.yaml'
+      - 'sample-workflows/**'
+  pull_request:
+    branches:
+      - 'main'
+    paths-ignore:
+      - 'docs/**'
+      - '.readthedocs.yaml'
+      - 'sample-workflows/**'
+  workflow_dispatch:
+    inputs:
+      push:
+        description: "Push source strings and translations to Transifex"
+        type: boolean
+      pull:
+        description: "Pull translations from Transifex"
+        type: boolean
+  schedule:
+    - cron: '0 0 * * 1'
+
+env:
+  TX_CLI_VERSION: '1.6.16'
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+
+    steps:
+      - name: Check out ${{ github.repository }}
+        uses: actions/checkout@v4.2.2
+
+      - name: Check out python-docs-theme
+        uses: actions/checkout@v4.2.2
+        with:
+          repository: python/python-docs-theme
+          persist-credentials: false
+          path: python-docs-theme
+
+      - name: Set up Python 3
+        uses: actions/setup-python@v5.6.0
+        with:
+          python-version: '3.12'
+          cache: 'pip'
+          cache-dependency-path: |
+            requirements.txt
+            python-docs-theme/requirements.txt
+
+      - name: Install Transifex CLI
+        working-directory: /usr/local/bin
+        run: |
+          curl -o- https://raw.githubusercontent.com/transifex/cli/master/install.sh | bash -s -- v${{ env.TX_CLI_VERSION }}
+
+      - name: Install dependencies
+        run: |
+          sudo apt update -y && sudo apt install gettext -y
+          pip3 install --upgrade pip
+          pip3 install -r requirements.txt -r python-docs-theme/requirements.txt
+
+        # required for generating .tx/config
+      - name: Generate POT files
+        working-directory: python-docs-theme
+        run: |
+          python babel_runner.py extract
+
+      - name: Generate Transifex configuration file
+        working-directory: python-docs-theme/python_docs_theme/locale/
+        run: |
+          sphinx-intl create-txconfig
+          sphinx-intl update-txconfig-resources --pot-dir . --locale-dir . --transifex-organization-name python-doc --transifex-project-name python-docs-theme
+          sed -i '/^minimum_perc *= 0$/s/0/1/' .tx/config
+          git diff .tx/config
```

Comment:
Using sphinx-intl don't seem to be necessary considering it is a single POT file. Considering this .tx/config file won't change, this could be stored in our python-docs-theme branch, and then simply `tx pull` from it.

This should do .tx/config:

```ini
[main]
host = https://www.transifex.com

[o:python-doc:p:python-docs-theme:r:python-docs-theme]
file_filter            = ./<lang>/LC_MESSAGES/python-docs-theme.po
source_file            = ../python-docs-theme/python_docs_theme/locale/python-docs-theme.pot
type                   = PO
minimum_perc           = 1
resource_name          = python-docs-theme
replace_edited_strings = false
keep_translations      = false
```

```suggestion
```